core/benches/banking_stage.rs

```diff
@@ -264,6 +267,7 @@ fn bench_banking(bencher: &mut Bencher, tx_type: TransactionType) {
             // in this chunk, but since we rotate between CHUNKS then
             // we should clear them by the time we come around again to re-use that chunk.
             bank.clear_signatures();
+            packet_deduper.reset();
```

Comment:
@taozhu-chicago here's the key part where we reset the banking stage's LRU cache in this benchmark that you had troubles with in your PR

Reply:
+1